cis/processor.py

```diff
@@ -71,16 +102,32 @@ def _validator_stage(self, kinesis_client=None):
                 profile_data=self.decrytped_profile
             ).retrieve_from_vault()
 
-        result = validation.Operation(
-            publisher=self.publisher,
-            profile_data=self.decrytped_profile,
-            user=self.user
-        ).is_valid()
+        if self._verify_signature():
+            logger.info(
+                'Signature verified successfully. Running additional validation plugins for user_id: {}'.format(
+                    self.decrytped_profile.get('user_id')
+                )
+            )
+
+            result = validation.Operation(
+                publisher=self.publisher,
+                profile_data=self.decrytped_profile,
+                user=self.user
+            ).is_valid()
+        else:
+            result = False
+
+        logger.info(
+            'Result of the validation operation for user_id: {} is {}'.format(
```

Comment:
'Signature verification failed' please - make it obvious

Reply:
Actually ... this logger is one indent out of "sig verification failed."  I'm adding another log statement to this block though that'll live in that else: on line 118